ceph_cfg/util_which.py

```diff
@@ -60,6 +60,7 @@ def fget(self):
 which_ceph_mon = memoise_which('ceph-mon')
```

Comment:
memoize is a standard computer term:

https://en.wikipedia.org/wiki/Memoization